addons/website/static/src/scss/website.scss

```diff
@@ -962,11 +962,21 @@ table.table_desc tr td {
     --o-carousel-indicators-dots-size: .75rem;
     --o-carousel-controllers-size: #{map-get($spacers, 5)};
```

Comment:
Question: why do we need to store the value in  a CSS variable and not in a SCSS one ? 🤔
As far as I can see this variable:
- it's not subjected to contextual changes
- it's not subjected to editor changes (eg. options)
- it's always used either directly  (eg 1) or in conjunction with a scss variable (eg. 2).

```scss
// 1
.o_carousel_controllers {
    height: var(--o-carousel-controllers-size);
}
// 2
.carousel-control-prev, .carousel-control-next {
    bottom: calc(var(--o-carousel-controllers-size) + #{map-get($spacers, 2)});
}

Reply:
To be honest, I thought using css variables was more optimal than using scss variables. But I guess since using css variables requires adding more calculation functions, it would be better to use scss variables, I'll change that.

Reply:
@stefanorigano 👋 
I've updated the code : https://github.com/odoo/odoo/compare/f826b7f5a772fd3ece7efa682c20856ddd4ff4fe..ade42186ea7696ff2719bc56e7a4d95689d239c7

What do you think ?